src/main/java/frc/robot/misc/Constants.java

```diff
@@ -23,6 +25,7 @@ public static final class Swerve {
         public static final double closedLoopRamp = 0.0;
 
         public static final double driveGearRatio = (6.75 / 1.0); // 6.75:1
+        public static final double turnGearRatio = (11.8 / 1.0);//(150.0 / 7.0);
```

Comment:
In general, it's a good idea to put a comment about 11.8 means, where it came from, how you chose it. Something to reference later if that value is off for any reason.